src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,102 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.InvalidDataException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
```

Comment:
additional tests can be added
```
register_christAgeUser_ok(){}
register_nullAgeUser_notOk(){}
register_lowAgeUser_notOk(){}
register_negativeAgeUser_notOk(){}
register_defaultValidUser_ok(){}
register_alreadyExistLoginUser_notOk(){}
register_nullUser_notOk(){}
register_nullLoginUser_notOk(){}
register_shortLoginUser_notOk(){}
register_nullPassUser_notOk(){}
register_shortPassUser_notOk(){}
```

Reply:
Thank you a lot, maybe not right understand tasks. End of week